app/views/saasbook/_table_of_contents.html.erb

```diff
@@ -9,6 +9,11 @@
       
     </a>
 </div>
+<form class="bd-search d-flex align-items-center" action="<%= search_path %>" method="get">
+		<i class="icon fas fa-search"></i>
+		<input type="search" class="form-control" name="q" id="search-input" placeholder="Search the docs ..." aria-label="Search the docs ..." autocomplete="off" >
+</form>
```

Comment:
Should we edit this to say "Search the textbook" instead of "search the docs"? "Search the docs" is the default text generated by SphinxSearch